test/mockContext.js

```diff
@@ -1,3 +1,6 @@
+/* eslint no-unused-vars: 1 */
```

Comment:
With the `globals` in gulp lint options, do we still need this `no-unused-vars: 1`?